### PR TITLE
Sabgit

### PR DESCRIPTION
**Pull request by sabgit**
Check return code from invoking the installer for both Chrome & Firefox drivers

### CODE REVIEW

File: postinstall [hunk 1]
```diff
@@ -1,17 +1,20 @@
 #!/bin/bash
 
+return_code=0
 if [[ -n "${BROWSER_DRIVER_INSTALLER_CHROME_VERSION:-}" && -n "${BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH:-}" ]]; then
     node ./index.js --browser-name chrome                                        \
                     --browser-version "$BROWSER_DRIVER_INSTALLER_CHROME_VERSION" \
-                    --target-path "$BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH"
+                    --target-path "$BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH" || return_code=$?
 else 
     echo "Environment variables for Chrome are not set, skipping post-install for Chrome"
```
Comment (thehcma) [inline postinstall:9]: The environment variables...

File: postinstall [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 fi
 
 if [[ -n "${BROWSER_DRIVER_INSTALLER_FIREFOX_VERSION:-}" && -n "${BROWSER_DRIVER_INSTALLER_GECKODRIVER_PATH:-}" ]]; then
     node ./index.js --browser-name firefox                                        \
                     --browser-version "$BROWSER_DRIVER_INSTALLER_FIREFOX_VERSION" \
-                    --target-path "$BROWSER_DRIVER_INSTALLER_GECKODRIVER_PATH"
+                    --target-path "$BROWSER_DRIVER_INSTALLER_GECKODRIVER_PATH" || return_code=$?
 else 
     echo "Environment variables for Firefox are not set, skipping post-install for Firefox"
```
Comment (thehcma) [inline postinstall:17]: same

File: postinstall [hunk 3]
```diff
@@ -1,17 +1,20 @@
 #!/bin/bash
 
+return_code=0
```
Comment (thehcma) [inline postinstall:3]: Does the failure produce a human-understandable error?

I checked and a think they do, but please revise them for grammar (most of them have issues), also use the same formatting we usually do (lower-case initial character for error messages as well as log messages).

**Review by thehcma — APPROVED** [review]

File: postinstall (hunk 2 above)
Comment (sabgit) [reply postinstall:17]: fixed

File: postinstall (hunk 1 above)
Comment (sabgit) [reply postinstall:9]: fixed

File: postinstall (hunk 3 above)
Comment (sabgit) [reply postinstall:3]: yes, it produces an error message like following:
```
Error: Failed to locate a version of chromedriver that matches the installed version of chrome (99). 
Valid chrome versions are: 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73
    at driverInstaller 
        ...
        throw new Error(
        ^

Error: Failed to locate a version of geckodriver that matches the installed version of firefox (88).
Valid firefox versions are: 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66 
 ...
```

Fixing capitalization...

**Pull request merged**